nextmv/nextmv/cloud/manifest.py

```diff
@@ -318,6 +322,43 @@ class ManifestPython(BaseModel):
     from the app bundle.
     """
 
+class ManifestOptionUI(BaseModel):
+    """
+    UI attributes for an option in the manifest.
+
+    You can import the `ManifestOptionUI` class directly from `cloud`:
+
+    ```python
+    from nextmv.cloud import ManifestOptionUI
+    ```
+
+    Parameters
+    ----------
+    control_type : str, optional
+        The type of control to use for the option in the Nextmv Cloud UI. This is
+        useful for defining how the option should be presented in the Nextmv
+        Cloud UI. Current control types include "input", "select", "slider", and
+        "toggle". This attribute is not used in the local `Options` class, but '
```

Comment:
Lost single-quote? :shrug: 

```suggestion
        "toggle". This attribute is not used in the local `Options` class, but
```